bundles/org.openhab.binding.draytonwiser/src/main/java/org/openhab/binding/draytonwiser/internal/handler/HotWaterHandler.java

```diff
@@ -132,7 +132,11 @@ private State getBoostRemainingState() {
             final Integer overrideTimeout = firstChannel.getOverrideTimeoutUnixTime();
 
             if (overrideTimeout != null && !"NONE".equalsIgnoreCase(firstChannel.getOverrideType())) {
-                return new QuantityType<Time>(overrideTimeout - (System.currentTimeMillis() / 1000L), SmartHomeUnits.SECOND).toUnit(SmartHomeUnits.MINUTE);
+                final QuantityType<Time> duration = new QuantityType<Time>(overrideTimeout - (System.currentTimeMillis() / 1000L), SmartHomeUnits.SECOND).toUnit(SmartHomeUnits.MINUTE);
```

Comment:
This was a bad idea. I see you now need to do the extra null check just to not get a compile error :sob:, while it doesn't add any value here. Besides the incorrect styling, I think you should just use seconds as unit and not convert it to minutes here. It would even be better to just divide by 60. But that misses the whole point of using smart units.